### PR TITLE
Rename filename comments

### PR DESCRIPTION
**Pull request by galanCA**
## Description

Renames the filename in the file description to keep consistency with the case of the filename. It also renames XMODULE in cmakelist to lower case. The author checked by doing a case sensitive search for all the files.

## Author Progress Checklist:

- [ ] Open draft pull request
    - [ ] Make title clearly understandable in a standalone change log context
    - [ ] Assign yourself the issue
    - [ ] Add at least one label (enhancement, bug, or maintenance)
    - [ ] Link the issue(s) addressed by this PR (under "Development" in the sidebar menu) 
- [ ] Make code changes (if you haven't already)
- [ ] Self-review of code
    - [ ] My code follows the style guidelines of this project
    - [ ] I have added comments to my code, particularly in hard-to-understand areas
    - [ ] I have only committed the necessary changes for this fix or feature
    - [ ] I have made corresponding changes to the documentation
    - [ ] My changes generate no new warnings
    - [ ] I have ensured that my fix is effective or that my feature works as intended by:
        - [ ] exercising the code changes in the test framework, and/or
        - [ ] manually verifying the changes (as explained in the the pull request description above)
    - [ ] My changes pass all local tests
    - [ ] My changes successfully passes CI checks
    - [ ] Add any unit test for proof and documentation.
    - [ ] Merge in main branch and address resulting conflicts and/or test failures.
- [ ] Move pull request out of draft mode and assign reviewers
- [ ] Iterate with reviewers until all changes are approved
    - [ ] Make changes in response to reviewer comments
    - [ ] Merge in main branch and address resulting conflicts and/or test failures.
    - [ ] Re-request review in GitHub

## Reviewer Checklist:

 - [ ] Read the pull request description
 - [ ] Perform a code review on GitHub
 - [ ] Confirm all CI checks pass and there are no build warnings
 - [ ] Pull, build, and run automated tests locally
 - [ ] Perform manual tests of the fix or feature locally
 - [ ] Add any review comments, if applicable
 - [ ] Submit review in GitHub as either
     - [ ] Request changes, or
     - [ ] Approve
 - [ ] Iterate with author until all changes are approved

### CODE REVIEW

File: src/cnguts.h [hunk 1]
```diff
@@ -194,7 +194,7 @@ extern SI cnRunSerial;			// incremented in cgInit; copied to Topi.runSerial in c
 extern VROUTINFO* UnspoolInfo;	// dm array of info re unspooling virtual reports into actual report files.
 								//  set up in cncult.cpp; format is for vrUnspool. vrpak.h struct.
 
-//---------------------- CGSOLAR.CPP PUBLIC VARIABLE ------------------------
+//---------------------- cgsolar.cpp PUBLIC VARIABLE ------------------------
```
Comment (nealkruis) [inline src/cnguts.h:197]: This is a tricky one. The file name is now consistent, but it is not consistent with the rest of the context here.

Comment (nealkruis) [reply src/cnguts.h:197]: I'm not saying it "needs" to be one way or the other. I'm just pointing out the new inconsistency you introduced and trying to understand why you came to the conclusion you did. I'm not saying it's wrong. Ultimately this specific case doesn't matter much at all. What really matters is that we are conscious of inconsistencies (even the more minor ones) and address them if possible.

Comment (tanaya-bigladder) [reply src/cnguts.h:197]: I think this is a non-issue compared to the fact that we have comments that say _PUBLIC VARIABLES_ and _FUNCTION DECLARATIONS_ at all.

File: src/cse.cpp [hunk 2]
```diff
@@ -3,7 +3,7 @@
 // that can be found in the LICENSE file.
 
 ///////////////////////////////////////////////////////////////////////////////
-// CSE.cpp : Main program module
+// cse.cpp : Main program module
```
Comment (tanaya-bigladder) [inline src/cse.cpp:6]: In our last iteration meeting, we touched upon the fact that the CSE executable name is in all caps, and maybe would remain that way. Does that argue for cse.h and cse.cpp to stay capitalized too? Obviously we can have CMake call the exe whatever we want, but in a simple build the file with `main()` would be the name of the binary.

**Comment by nealkruis** [comment]
@galanCA there are unresolved comments in this PR. Do you plan to make any changes still?

**Review by nealkruis — APPROVED** [review]

**Pull request merged**